pkg/cloudprovider/metal/instances_v2_test.go

```diff
@@ -252,6 +260,103 @@ var _ = Describe("InstancesV2", func() {
 	})
 })
 
+var _ = Describe("InstancesV2", func() {
```

Comment:
Maybe we should indicate in the description that this is an alternative instances controller.